snap/snapcraft.yaml

```diff
@@ -292,8 +293,22 @@ parts:
       export CGO_LDFLAGS="-L${CRAFT_STAGE}/lib/ -L${CRAFT_STAGE}/usr/local/lib/"
       export CGO_LDFLAGS_ALLOW="(-Wl,-wrap,pthread_create)|(-Wl,-z,now)"
 
+      # Determine OVN package version
+      pkg_version=$(
+          apt-cache policy ovn-host | \
```

Comment:
See comment above, I'd rather et this from the actual binary consumed by the build.